### DISCUSSION TITLE
Trophies

### DISCUSSION THREAD

**Discussion by shibedroid** [cwendt94/espn-api#340] [Ideas]
Have you found any information regarding trophies from leagues (fantasy football) existing when trophies were available?

I would love to recover the data, even if only the name of the trophies.

**Comment by cwendt94** [comment]
By trophies, like the ones on the main league page that has 1st 2nd and 3rd place?

**Reply by shibedroid** [reply]
Nope, the custom trophies that you could create on the site and could be awarded manually or automatically, like "Most points scored during the season". It was a League Manager feature.

**Reply by cwendt94** [reply]
Oh I don't think I have seen that before. Is your league public? I could check out the data. If not I could give you the API call that will return all of your leagues data and I can take a look at the response to see if trophies are in it.

**Reply by shibedroid** [reply]
My league is not public. It could work with the API call, I can send you the result. Thanks

**Reply by cwendt94** [reply]
Awesome, here is the API you can use. If you are logged in to your ESPN account you can just put it in the browser and not have to worry about credentials
`https://fantasy.espn.com/apis/v3/games/ffl/seasons/${year}/segments/0/leagues/${leagueId}?view=mTeam&view=mRoster&view=mMatchup&view=mStandings`

Make sure to replace `${year}` and `${leagueId}` with your own.
